### ISSUE TITLE
Lack of support for applying extra spacing between Japanese and Western text

### ISSUE THREAD

**Issue by xfq** [w3c/jlreq#312]
<i class="meta">This issue is applicable to Chinese and Japanese.</i>

In Japanese composition, it is usually recommended to apply extra spacing between Japanese and Western text to increase the recognition of Western text.

Currently, browsers do not support this feature, so content authors have to work around the problem by adding spaces manually.

<b class="meta">For more details, see [this GitHub issue](https://github.com/w3c/clreq/issues/401), which is being used to track this gap.  Please discuss this issue there.</b>

**Comment by xfq** [comment]
_The first comment in this issue contains text that will automatically appear in one or more gap-analysis documents as a subsection with the same title as this issue. Any edits made to that comment will be immediately available in the document. Proposals for changes or discussion of the content can be made in comments below this point._

_Relevant gap analysis documents include:_
_[**Chinese**](https://w3c.github.io/clreq/gap-analysis/#issue401_spacing) • [Japanese](https://w3c.github.io/jlreq/gap-analysis/#issue312_spacing)_

**Comment by frivoal** [comment]
There exists a css-property to deal with that in css-text-4: https://drafts.csswg.org/css-text-4/#text-spacing-property

`text-spacing: ideograph-alpha ideograph-numeric`

I don't believe it to be implemented anywhere yet, though.

**Comment by xfq** [comment]
@frivoal Yeah, I also mentioned it in https://github.com/w3c/clreq/issues/401 (although I didn't mention `ideograph-numeric`). I think the current CSS spec has at least two potential issues:

* First, this spacing is currently not customizable. See https://github.com/w3c/jlreq/issues/163 for more details.
* Second, `text-spacing` does not currently support non-CJK languages. See the thread in https://github.com/w3c/clreq/issues/401#issuecomment-947220001

**Comment by r12a** [comment]
Hmm. This discussion would be better held at https://github.com/w3c/clreq/issues/401, since it is the issue which is being tracked by the gap pipeline.  Implementers using the pipeline may not see this thread.

(Perhaps the boilerplate text should indicate where to hold discussions.)

**Comment by MurakamiShinyu** [comment]
FYI,
> There exists a css-property to deal with that in css-text-4: https://drafts.csswg.org/css-text-4/#text-spacing-property
> 
> `text-spacing: ideograph-alpha ideograph-numeric`
> 
> I don't believe it to be implemented anywhere yet, though.

`text-spacing: ideograph-alpha ideograph-numeric` is being implemented in Vivliostyle very recently.

- https://github.com/vivliostyle/vivliostyle.js/pull/814
